cg/store/api/find_business_data.py

```diff
@@ -316,6 +313,16 @@ def get_sample_by_customer_and_name(
             name=sample_name,
         ).first()
 
+    def get_number_of_reads_for_sample_from_metrics(self, sample_internal_id: str) -> int:
```

Comment:
Neat 👍  Have we double checked on existing samples that this is correct, ie that we're not missing a factor of 2 somewhere?

Reply:
Yes, me and @ChrOertlin did some validation when we implemented the parsing and calculation of the sequencing metrics. The read counts in the metrics table are consistent with the ones previously reported in `cgstats`

Reply:
I've verified that the generated sample counts are the sum of the read counts in the sequencing metrics table. We need to patch the data in that table though, many flow cells and samples seem to be missing in status db compared to cg stats.